statefun-sdk-python/README.md

```diff
@@ -17,7 +17,7 @@ to get yourself started with Stateful Functions in Python.
 
 For a fully detailed documentation, please visit the [official docs](https://ci.apache.org/projects/flink/flink-statefun-docs-master).
 
-For code examples, please take a look at the [examples](../statefun-examples/).
+For code examples, please take a look at the [examples](https://github.com/apache/flink-statefun-playground/tree/dev/python).
```

Comment:
Can that not be pointing to `dev` but rather to the main branch, or the latestest released branch?

Reply:
I think the latest release branch makes sense since it is also what is shown for a specific PyPI package.

Reply:
What is not so optimal is that updating these links is currently a manual process. As a follow-up we might wanna think about how this could be automated by establishing a branch mapping between `flink-statefun` and `flink-statefun-playground`.